Source/WebCore/rendering/svg/RenderSVGResourceMarkerInlines.h

```diff
@@ -38,19 +43,19 @@ FloatPoint RenderSVGResourceMarker::referencePoint() const
 
 std::optional<float> RenderSVGResourceMarker::angle() const
 {
-    if (markerElement().orientType() == SVGMarkerOrientAngle)
-        return markerElement().orientAngle().value();
+    if (protectedMarkerElement()->orientType() == SVGMarkerOrientAngle)
```

Comment:
This could be stored so we don't create two temporaries of protectedMarkerElement()
if (Ref markerElement = this->markerElement; markerElement->orientType() == SVGMarkerOrientAngle)

Reply:
I don't think I touched this code, but makes sense indeed. Done.